sensa/experiment/remote.py

```diff
@@ -39,9 +39,9 @@
 
 class ExperimentClient(_Client):
     """
-    A client for the `Cortex experiment and model management API <https://cognitivescale.github.io/cortex-fabric/docs/models/experiments>`_. You can find a pre-created instance of this class on every :class:`cortex.client.Client` instance via the :attr:`Client.experiments` attribute.
+    A client for the `Cortex experiment and model management API <https://cognitivescale.github.io/cortex-fabric/docs/models/experiments>`_. You can find a pre-created instance of this class on every :class:`sensa.client.Client` instance via the :attr:`Client.experiments` attribute.
 
-    >>> from cortex.client import Cortex; client = Cortex.client();
+    >>> from sensa.client import Sensa; client = Cortex.client();
```

Comment:
`Cortex.client();` -> `Sensa.client();`

There are a other instances in this module